readme.md

```diff
@@ -170,9 +170,18 @@ See [the demo](https://github.com/yomotsu/camera-movement-comparison#dolly-vs-zo
 | `.colliderMeshes`         | `array`   | `[]`        | An array of Meshes to collide with camera ¹. |
 | `.infinityDolly`          | `boolean` | `false`     | `true` to enable Infinity Dolly ². |
 | `.restThreshold`          | `number`  | `0.0025`    | Controls how soon the `rest` event fires as the camera slows |
+| `.minZoom` 	            | `number`  | `int`       | Limit for camera zoom pos/neg |
+| `.maxZoom` 	            | `number`  | `int`       | Limit for camera zoom pos/neg |
 
 1. Be aware colliderMeshes may decrease performance. Collision test uses 4 raycasters from camera, since near plane has 4 corners.
 2. When the Dolly distance less than the minDistance, the sphere of radius will set minDistance.
+3. Every 360 degrees are added to the value. `360º = 360 * THREE.MathUtils.DEG2RAD = 6.283185307179586`, `720 = 12.566370614359172`. To restart use this function normalizeRotations() or this code 
+```js
+const azi = 0;
+cameraController.addEventListener("control", (_event) => {
+	azi = _event.target.azimuthAngle % (360 * THREE.MathUtils.DEG2RAD);
+}, false);
+```
```

Comment:
when you rotate to the side `.azimuthAngle` is summing 
0 -> 360º -> 720º -> 1080º + 45º = 1125º
now if you use `rotateTo()` camera spinning back all 4x right? okey if you use this code 
`.azimuthAngle =  .azimuthAngle % (360 * THREE.MathUtils.DEG2RAD)` you are reset 4x turns to 0 + 45º
now if you execute `rotateTo()` camera rotate only -45º to 0

Reply:
That's right.
However, I think, the code example doesn't solve the situation, because `azi` is just a variable (also it should not be a `const`?).
Maybe the example could be omitted at this time?

Reply:
Hmm yes you right, ok no problem I change it, omitting this sample

Reply:
maybe can add new parameter with this function `.azimuthAbsoluteAngle` what you thing about it?

Reply:
Thanks for understanding.

> maybe can add new parameter 

Thanks for your suggestion.
Actually, It can be done easily with [`THREE.MathUtils.euclideanModulo`](https://threejs.org/docs/index.html?q=math#api/en/math/MathUtils.euclideanModulo) in user-land, I think.

e.g.
```js
THREE.MathUtils.euclideanModulo( -2890, 360 ); // 350
```

Reply:
I think people do not know the existence of many of the functions, which integrates in Three for that reason to be confused when using the plugin. Or they just do not understand the hierarchy of elements. You should put plugin for user Mid / Pro Level XD

Reply:
Thanks for considering the users😊
We could add an explanation of how to normalize the angle on the user's side with `THREE.MathUtils.euclideanModulo`.
Maybe like the below:

If you need a normalized azimuth angle (between 0 and 360 deg), compute with `THREE.MathUtils.euclideanModulo`.
e.g.
```js
const normalizedAzimuthAngle = THREE.MathUtils.euclideanModulo( cameraControls.azimuthAngle, 360 * THREE.MathUtils.DEG2RAD );
```

However, regarding adding the helper on the library side, That must be done with another github issue or PR.

Reply:
You are welcome :)